### PR TITLE
[DUBBO-9]add apache-rat-plugin to check asf license header

### PR DESCRIPTION
**Pull request by lovepoem**
## What is the purpose of the change

add apache-rat-plugin to check asf license header for https://issues.apache.org/jira/browse/DUBBO-9

As discussed in the [mail list](https://lists.apache.org/list.html?dev@dubbo.apache.org:2018-3:How%20to%20add%20Apache%20license%20head%20to%20the%20files%20of%20hessian-lite%20sub%20project),the hessian_lite sub-project is  excluded from check 

## Brief changelog

add missed asf license headers

## Verifying this change

CI pass

### CODE REVIEW

File: hessian-lite/src/test/java/com/alibaba/com/caucho/hessian/io/LocaleSerializerTest.java [hunk 1]
```diff
@@ -1,3 +1,19 @@
+/*
```
Comment (htynkn) [inline hessian-lite/src/test/java/com/alibaba/com/caucho/hessian/io/LocaleSerializerTest.java]: should hessian_lite be excluded from asf header check?

File: pom.xml [hunk 2]
```diff
@@ -443,6 +444,56 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```
Comment (htynkn) [inline pom.xml]: in checkstyle, there is one rule to check asf header. If team decide to use apache rat plugin, we may need remove checkstyle rule. 
Related files:
+ https://github.com/apache/incubator-dubbo/blob/master/codestyle/checkstyle-header.txt
+ https://github.com/apache/incubator-dubbo/blob/master/codestyle/checkstyle.xml#L10

File: hessian-lite/src/test/java/com/alibaba/com/caucho/hessian/io/LocaleSerializerTest.java (hunk 1 above)
Comment (lovepoem) [reply hessian-lite/src/test/java/com/alibaba/com/caucho/hessian/io/LocaleSerializerTest.java]: >should hessian_lite be excluded from asf header check?

Yes, this is discussed in the dev@a

File: pom.xml (hunk 2 above)
Comment (lovepoem) [reply pom.xml]: ok

**Comment by ralf0131** [comment]
Looks good to me!

**Pull request merged**